tests/api/runtimes/test_spark.py

```diff
@@ -115,6 +118,28 @@ def _assert_volume_and_mounts(
                 == {}
             )
 
+    def _assert_driver_and_executor(self, body, expected_values):
```

Comment:
Should probaby be named `_assert_driver_and_executor_resources` or something like that.